src/Views/EncryptView.vala

```diff
@@ -44,21 +44,21 @@ public class EncryptView : AbstractInstallerView {
         title_label.get_style_context ().add_class ("h2");
         title_label.valign = Gtk.Align.START;
 
-        var protect_image = new Gtk.Image.from_icon_name ("emoji-body-symbolic", Gtk.IconSize.LARGE_TOOLBAR);
+        var protect_image = new Gtk.Image.from_icon_name ("security-high-symbolic", Gtk.IconSize.LARGE_TOOLBAR);
 
         var protect_label = new Gtk.Label (_("Encrypting this drive protects data from being read by others with physical access to this device."));
         protect_label.max_width_chars = 52;
         protect_label.wrap = true;
         protect_label.xalign = 0;
 
-        var performance_image = new Gtk.Image.from_icon_name ("emoji-objects-symbolic", Gtk.IconSize.LARGE_TOOLBAR);
+        var performance_image = new Gtk.Image.from_icon_name ("sensors-voltage-symbolic", Gtk.IconSize.LARGE_TOOLBAR);
 
         var performance_label = new Gtk.Label (_("Drive encryption may minimally impact read and write speed when performing intense tasks."));
         performance_label.max_width_chars = 52;
         performance_label.wrap = true;
         performance_label.xalign = 0;
 
-        var restart_image = new Gtk.Image.from_icon_name ("rotation-allowed-symbolic", Gtk.IconSize.LARGE_TOOLBAR);
+        var restart_image = new Gtk.Image.from_icon_name ("rotation-locked-symbolic", Gtk.IconSize.LARGE_TOOLBAR);
```

Comment:
`system-restart-symbolic` makes a lot more sense here

Reply:
I liked the rotation lock symbolic as it visually indicated a lock. While it's less semantic, I felt that better conveyed the correct meaning behind the phrase it accompanied. Additionally, it seems to have better coverage with various icon themes; `system-restart-symbolic` doesn't appear to be present outside of the Pop icon theme. I can still change it, just feedback.

Reply:
>I liked the rotation lock symbolic as it visually indicated a lock. While it's less semantic, I felt that better conveyed the correct meaning behind the phrase it accompanied. 

The meaning is just supposed to be "restart", but I see what you mean with the lock. In that case, we'd probably want to come up with a semantic name in the icon theme and symlink it instead of abusing icon names.

>Additionally, it seems to have better coverage with various icon themes; system-restart-symbolic doesn't appear to be present outside of the Pop icon theme.

Eh, not a convincing rationale because 1. we're building our fork of the installer for a very controlled environment with a known stylesheet and icon set, and 2. there are zero guarantees that `rotation-locked-symbolic` would convey the same idea in other icon sets.

If we go more semantic in our icon theme, I'd recommend something like `system-restart-locked-symbolic` or `system-restart-encrypted-symbolic`. It's in the style of the more common icons and should fall back properly in other icon sets if we care.

Reply:
In the interest of fixing the pull request before a public ISO is released (because someone can't wait), I changed it to `system-restart-symbolic` in #16.